tests/components/egps/conftest.py

```diff
@@ -0,0 +1,61 @@
+"""Configure tests for egps."""
+from collections.abc import Generator
+from typing import Final
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from pyegps.fakes.powerstrip import FakePowerStrip
+import pytest
+
+from homeassistant.components.egps.const import CONF_DEVICE_API_ID, DOMAIN
+from homeassistant.const import CONF_NAME
+
+from tests.common import MockConfigEntry
+
+DEMO_CONFIG_ENTRY: Final = {
```

Comment:
getting nitpicky here, but since this is not really the config enty, I'd prefer `DEMO_DATA` or something along that lines